docs/source/developer/translation.rst

```diff
@@ -0,0 +1,128 @@
+Translation
+##############################################################################
+
+
+Translation Principles
+=============================================================================
+
+The following principles were used to guide the design of a translation system for CS Field Guide:
+
+1. Translations should be 'first class citizens' - they should exist as independent entities from the source content, and be coupled as loosely as possible to it.
+2. Translated content should remain accessible even if the source material changes, and an updated translation is not yet available.
+3. Users should be able to see all content that exists, even if it is not available in their language.
+4. Translated content should be presented solely in the users language, without being interspersed with English.
+
+
+Translatable Files
+=============================================================================
+Translatable content is stored in five types of files:
+
+- Markdown files, for content to be processed by Verto.
+- YAML files containg field translations for a given model type.
+- HTML templates.
+- Python code.
+- JavaScript code.
+
+The first two types of file are stored inside the directory tree for a given language (ie. the directory named using the Django locale code).
+
+.. note::
+
+  YAML translation files must contain translations for only one model type, but may contain translations for multiple instances of that model, and for multiple fields.
+
+  They are structured as follows
+
+  .. code-block:: yaml
+
+    <object-slug-1>:
+        <field-1>: <translated value for field-1>
+        <field-2>: <translated value for field-2>
+    <object-slug-2>
```

Comment:
I've never seen this syntax, or a YAML file with translations in general, are we sure it's used?

Reply:
Yeah this is all completely wrong. It's all related to CS Unplugged....